Makefile

```diff
@@ -255,22 +256,20 @@ bundle-build: ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(MAKE) docker-push OPERATOR_IMAGE=$(BUNDLE_IMG)
 
-.PHONY: fix-csv-replaces
-fix-csv-replaces: $(YQ)
+.PHONY: set-replaces-file
+set-replaces-file: $(YQ)
 	$(eval REPLACES_VERSION=$(shell curl -sSL -H "Accept: application/vnd.github+json" \
                https://api.github.com/repos/Kuadrant/authorino-operator/releases/latest | \
                jq -r '.name'))
-	V="authorino-operator.$(REPLACES_VERSION)" $(YQ) eval '.spec.replaces = strenv(V)' -i bundle/manifests/authorino-operator.clusterserviceversion.yaml
+	V="authorino-operator.$(REPLACES_VERSION)" $(YQ) -n '.config.replaces = strenv(V)' > $(BUNDLE_CONFIG_FILE)
 
 .PHONY: prepare-release
 prepare-release:
+	$(MAKE) set-replaces-file
```

Comment:
Not sure if I like that the config file is only created when we fetch the latest release. What if I want one only for the default Authorino image option?

WDYT about:

1. `$(MAKE) create-config-file` (or `$(MAKE) create-build-file`, depending on what you choose at https://github.com/Kuadrant/authorino-operator/pull/161/files#r1403594058)
  1.1. `@echo "config:" > $(BUNDLE_CONFIG_FILE)`
  1.2. `$(MAKE) set-authorino-default-image`
  1.3. `$(MAKE) set-replaces-directive`
2. `$(MAKE) manifests bundle …`

?

Reply:
Great suggestions, made this change